README.md

```diff
@@ -14,107 +14,6 @@ Additionally, there are [several tools](https://maplibre.org/martin/tools.html)
 
 See [Martin book](https://maplibre.org/martin/) for complete documentation.
 
-## Installation
```

Comment:
I think we should create some something like a list of important information, and point it to the book - e.g. installation should point directly to the installation section, etc.  Just the most important topics, not everything?

Reply:
Like this?

## Helpful Martin docs
* [Installation](https://maplibre.org/martin/installation.html)
* Running with [CLI](https://maplibre.org/martin/run-with-cli.html) or [configuration file](https://maplibre.org/martin/config-file.html)
* [Usage and API](https://maplibre.org/martin/using.html)

Reply:
Yes, but without the header - just a few links.  We may also want to add a "quick start" - makes novice users go there first